src/lib.js

```diff
@@ -244,19 +253,35 @@ function parseBankAccounts($) {
       },
       type: {
         sel: 'a.account--name',
-        attr: 'class',
-        parse: helpers.getAccountTypeFromCSS
+        attr: 'href',
+        parse: helpers.getAccountTypeFromUrl
+      },
+      url: {
+        sel: 'a.account--name',
+        attr: 'href',
+        parse: href => baseUrl + href
       }
     },
     'table.table--accounts tr.table__line--account'
   )
 
+  for (let account of accounts) {
+    const $ = await request({ uri: account.url })
```

Comment:
Account numbers are available in the CSV. Why do you make this request?

If this request is legitimate, you can include the lines 280 and 281 in this loop to refactor the code and avoid a unnecessary loop. :)

Reply:
I need it to get the account number of an empty account (with no operations). I stumbled upon this case when retrieving only the data for one year of a dormant account. 
It's also necessary for accounts such as life-insurance, where there is no operation to retrieve.

I tried to include it in the `forEach` loop, but apparently it doesn't support async operations. If you have a suggestion to include it, I'm all ears!

Reply:
Indeed, you're right! :) 

It is better (at least easier) to avoid `forEach` for async functions.